GitUI/UserControls/RevisionGrid/Layouts/ILayoutRenderer.cs

```diff
@@ -0,0 +1,83 @@
+namespace GitUI.UserControls.RevisionGrid.Layouts
+{
+    public interface ILayoutRenderer
+    {
+        int Height { get; }
+        bool IsCardLayout { get; }
+        bool IsFilledBranchesLayout { get; }
+        bool IsGraphLayout { get; }
+        RevisionGridLayout Layout { get; }
+    }
+
+    public sealed class FilledBranchesSmallLayoutRenderer : ILayoutRenderer
+    {
+        public int Height => -1; // TODO: this is dynamically calculated
```

Comment:
Is TODO a mark that you plan to do anything? It seems intended, so mybe just remove it